packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperPlotNav.tsx

```diff
@@ -262,12 +262,12 @@ const QueryWrapperPlotNav: React.FunctionComponent<QueryWrapperPlotNavProps> = (
     initQueryJson,
     showLastUpdatedOn,
     unitDescription,
-    additionalFiltersLocalStorageKey,
+    additionalFiltersSessionStorageKey,
   } = props
 
   const entityId = parseEntityIdFromSqlStatement(sql)
   const additionalFilters = getAdditionalFilters(
-    additionalFiltersLocalStorageKey ?? entityId,
```

Comment:
Instead of falling back to looking for an entry using the entity ID, just don't look for a filter unless the additional filter key is supplied.